adafruit_pyportal.py

```diff
@@ -829,7 +832,23 @@ def show_QR(self, qr_data, qr_size=1, x=0, y=0):  # pylint: disable=invalid-name
         self._qr_group.x = x
         self._qr_group.y = y
         self._qr_group.append(qr_sprite)
-        board.DISPLAY.show(self._qr_group)
+        if hide_background:
+            board.DISPLAY.show(self._qr_group)
+            self._qr_only = True
+        else:
+            self._qr_only = False
```

Comment:
One style nitpick here is what was taught to me as "boolean zen", if you set True and False in matching branches you can do a direct assignment instead. Here that would be `self._qr_only = hide_background` after the if portion.

Reply:
That's really great. Love that tip. I've updated it and pushed a new commit.